components/dashboard/src/service/service.tsx

```diff
@@ -153,6 +153,7 @@ function testPublicAPI(service: any): void {
                             timeoutMs: 7 * 60 * 1000,
                         },
                     )) {
+                        console.log("Reply: " + reply.reply);
```

Comment:
it is enabled in prod, so can be noisy, maybe use another level? I was using Chrome debugger to verify it.

Reply:
Done